lib/react-archer.js

```diff
@@ -0,0 +1 @@
){if(!(t instanceof e))throw new TypeError("Cannot call a class as a function")}(this,e);for(var n=arguments.length,o=new Array(n),i=0;i<n;i++)o[i]=arguments[i];return R(M(M(r=function(t,e){return!e||"object"!==j(e)&&"function"!=typeof e?M(t):e}(this,(t=S(e)).call.apply(t,[this].concat(o))))),"registerTransitions",function(t){var e=r.generateSourceToTarget(t);if(!r.props.context.registerTransitions)throw new Error('Could not find "registerTransition" in the context of <ArcherElement>. Wrap the component in a <ArcherContainer>.');r.props.context.registerTransitions(e)}),R(M(M(r)),"generateSourceToTarget",function(t){var e=r.props.id;return t.map(function(t){var r=t.id,n=t.sourceAnchor,o=t.targetAnchor,i=t.label,a=t.style;return{source:{id:e,anchor:n},target:{id:r,anchor:o},label:i,style:a}})}),R(M(M(r)),"unregisterTransitions",function(){if(!r.props.context.unregisterTransitions)throw new Error('Could not find "unregisterTransitions" in the context of <ArcherElement>. Wrap the component in a <ArcherContainer>.');r.props.context.unregisterTransitions(r.props.id)}),R(M(M(r)),"onRefUpdate",function(t){if(t){if(!r.props.context.registerChild)throw new Error('Could not find "registerChild" in the context of <ArcherElement>. Wrap the component in a <ArcherContainer>.');r.props.context.registerChild(r.props.id,t)}}),R(M(M(r)),"unregisterChild",function(){if(!r.props.context.unregisterChild)throw new Error('Could not find "unregisterChild" in the context of <ArcherElement>. Wrap the component in a <ArcherContainer>.');r.props.context.unregisterChild(r.props.id)}),r}return function(t,e){if("function"!=typeof e&&null!==e)throw new TypeError("Super expression must either be null or a function");t.prototype=Object.create(e&&e.prototype,{constructor:{value:t,writable:!0,configurable:!0}}),e&&P(t,e)}(e,o.a.Component),function(t,e,r){e&&C(t.prototype,e),r&&C(t,r)}(e,[{key:"componentDidUpdate",value:function(t){a()(t.relations,this.props.relations)||this.registerTransitions(this.props.relations)}},{key:"componentDidMount",value:function(){0!==this.props.relations.length&&this.registerTransitions(this.props.relations)}},{key:"componentWillUnmount",value:function(){this.unregisterChild(),this.unregisterTransitions()}},{key:"render",value:function(){return o.a.createElement("div",{style:function(t){for(var e=1;e<arguments.length;e++){var r=null!=arguments[e]?arguments[e]:{},n=Object.keys(r);"function"==typeof Object.getOwnPropertySymbols&&(n=n.concat(Object.getOwnPropertySymbols(r).filter(function(t){return Object.getOwnPropertyDescriptor(r,t).enumerable}))),n.forEach(function(e){R(t,e,r[e])})}return t}({},this.props.style,{position:"relative"}),className:this.props.className,ref:this.onRefUpdate},this.props.children)}}]),e}();R(W,"defaultProps",{relations:[]});var L=function(t){return o.a.createElement(x,null,function(e){return o.a.createElement(W,A({},t,{context:e}))})};r.d(e,"ArcherElement",function(){return L}),r.d(e,"ArcherContainer",function(){return k})}])});
```

Comment:
Did you intend to commit this built file?

Reply:
No that's my bad. This was much easier to test with as opposed to a private npm publish flow